docs/sources/next/javascript-api/k6-browser/elementhandle/setchecked.md

```diff
@@ -0,0 +1,69 @@
+---
+title: 'setChecked(checked[, options])'
+description: 'Browser module: elementHandle.setChecked method'
+---
+
+# setChecked(checked[, options])
+
+{{< admonition type="warning" >}}
+
+Use [`locator.setChecked(checked [, options])`](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-browser/locator/setchecked/) instead.
```

Comment:
Not required, but I think it would be nice to provide a reason why the user should be using the `locator.setChecked` instead.

Reply:
Good call 👍 We have a lot of these in other places as well. It'd make more sense to do them all in one go. Do you think it makes sense to create an issue for this?

Reply:
Yeah, that makes sense to me!

Reply:
Here is the issue: https://github.com/grafana/k6-docs/issues/1710